integration_tests/run_pyspark_from_build.sh

```diff
@@ -174,7 +174,16 @@ else
     else
         TEST_PARALLEL_OPTS=("-n" "$TEST_PARALLEL")
     fi
-    RUN_DIR=${RUN_DIR-"$SCRIPTPATH"/target/run_dir}
+
+    # Delete old run directories
```

Comment:
I'm not sure it's in the scope of this script to remove other directories.  The run directory is already in a `target/` directory which typically gets cleaned out via `mvn clean` anyway.

Reply:
good point. although I personally often don't run the tests from maven.

Reply:
Yeah I never run the test from Maven either.  However `mvn clean` does get run a fair bit by every developer which will clean out these default directories.  As for externally configured directories, I'm even more skeptical we should try to do any cleanup.  For example, if someone configured a run directory that was also used for other things, we could do some "surprising" deletes there that the user would not be happy about.

Reply:
Thank you for the discussion! I removed the cleanup code.